builder/googlecompute/driver_gce.go

```diff
@@ -215,6 +215,25 @@ func (d *driverGCE) RunInstance(c *InstanceConfig) (<-chan error, error) {
 		return nil, err
 	}
 
+	// Subnetwork
+	// Validate Subnetwork config now that we have some info about the network
+	if !network.AutoCreateSubnetworks && len(network.Subnetworks) > 0 {
+		// Network appears to be in "custom" mode, so a subnetwork is required
+		if c.Subnetwork == "" {
+			return nil, fmt.Errorf("a subnetwork must be specified")
```

Comment:
I'm not sure how to add a test for this one, this check is based on attributes retrieved from the Google API for the network. Would you suggest I mock out the API calls?